File: 0796-rotate-string/0796-rotate-string.java
class Solution {
    public boolean rotateString(String s, String goal) {
        if(s.length()!=goal.length()){
            return false;
        }
       StringBuilder him=new StringBuilder(s);
       for(int i=0;i<s.length();i++){
        if(him.toString().equals(goal)){
            return true;
        }
        char ch=him.charAt(0);
        him.deleteCharAt(0);

        him.append(ch);
       }
       return false;
    }
}

File: 0796-rotate-string/0796-rotate-string.py
class Solution:
    def rotateString(self, s: str, goal: str) -> bool:
        if len(s)!=len(goal):
            return False
        sb=list(s)
        for i in range(len(s)):
            if ''.join(sb)==goal:
                return True
            p=sb[0]
            sb.pop(0)
            sb.append(p)
        return False
        